ruby/engine/embedded_gems.rb

```diff
@@ -0,0 +1,281 @@
+#(const std::vector<openstudio::path>& includeDirs, std::vector<openstudio::path> gemPathDirs,
+#                               const openstudio::path& gemHomeDir, const openstudio::path& bundleGemFilePath,
+#                               const openstudio::path& bundleGemDirPath, const std::string& bundleWithoutGroups) {
+
+def setup_embedded_gems() #includeDirs, gemPathDirs, gemHomeDir, bundleGemFilePath, bundleGemDirPath, use_bundler, bundleWithoutGroups)
+
+  # Unset RUBY ENVs if previously set (e.g. rvm sets these in shell)
+
+  ENV.delete('GEM_HOME') if ENV['GEM_HOME']
+  ENV.delete('GEM_PATH') if ENV['GEM_PATH']
+  ENV.delete('BUNDLE_GEMFILE') if ENV['BUNDLE_GEMFILE']
+  ENV.delete('BUNDLE_PATH') if ENV['BUNDLE_PATH']
+  ENV.delete('BUNDLE_WITHOUT') if ENV['BUNDLE_WITHOUT']
+
+  if !includeDirs.empty?
```

Comment:
This file is unused... I originally thought about doing more in ruby, but I finally decided to break it into several C++ functions in bindings/InitRubyBindings, some of which are deferring to ruby eval, others are using lower level ruby C API functions where appropriate

Reply:
I should probably delete it